Islandora-CLAW/Crayfish
# Issue 565

## GitHub Issue

Part of:
https://github.com/Islandora-CLAW/CLAW/issues/565

Depends on: 
https://github.com/Islandora-CLAW/Crayfish-Commons/pull/5

## What does this Pull Request do?

This updates Houdini to use the new JWT service provided in https://github.com/Islandora-CLAW/Crayfish-Commons/pull/5. It does this by using the new `IslandoraServiceProvider` that is also provided in that pull request. That makes the app setup very simple for this, which I think it what we want. After this goes in the other services should be updated to use the `IslandoraServiceProvider`.

## How should this be tested?

- Pull in https://github.com/Islandora-CLAW/Crayfish-Commons/pull/5
- Setup your composer so that this uses that pull request
- Make sure Houdini still works
- Make sure you can enable / disable logging with the configuration
- Make sure you can enable / disable security with the configuration file

## Interested parties
@Islandora-CLAW/committers
